lib/aptible/cli/subcommands/logs.rb

```diff
@@ -25,6 +28,147 @@ def logs
               ENV['ACCESS_TOKEN'] = fetch_token
               exit_with_ssh_portal(op, '-o', 'SendEnv=ACCESS_TOKEN', '-T')
             end
+
+            desc 'logs_from_archive --bucket NAME --region REGION ' \
+                 '--stack NAME --decryption-keys ONE [OR MORE] ' \
+                 '--download-location [ [ --string-matches ONE [OR MORE] ] ' \
```

Comment:
Missing placeholder in the description

```suggestion
                 '--download-location LOCATION [ [ --string-matches ONE [OR MORE] ] ' \
```

Reply:
:+1: